cmd/minikube/cmd/start.go

```diff
@@ -235,6 +319,10 @@ func clusterUp(config *cluster.MachineConfig) {
 			if exists {
 				key = minishiftToClusterUp[key]
 			}
+			if !ocSupportFlag(cmdName, key) {
```

Comment:
as we discussed, it would be much nicer for the user to fail earlier. So I'd have preferred to first download oc and do some validation, but let's go with this for now. Hopefully once we align  with 1.5 not so many people will run into this anyways.